text/src/autogluon/text/automm/configs/model/fusion_mlp_image_text_tabular.yaml

```diff
@@ -62,11 +62,13 @@ model:
     insert_sep: True
     text_segment_num: 2
     stochastic_chunk: False
-    text_detection_length: 10                          # for augmentation purpose: naive way to different text column versus tabular column that are treated as text
+    text_aug_detect_length: 10                          # for augmentation purpose: naive way to different text column versus tabular column that are treated as text
```

Comment:
We may need to improve the comment of this key: 
We perform text augmentation only if a text has more than `text_detection_length ` words. It is used to differentiate text columns versus tabular columns that are treated as text.